todo_app/templates/index.html

```diff
@@ -1,18 +1,104 @@
 {% extends "layout.html" %}
-{% block title %}To-Do App{% endblock %}
+{% block title %} [to_do || !to_do] app {% endblock %}
 
 {% block content %}
+<style type="text/css">
+  .jumbotron{
+    background-image: url('https://i.ibb.co/JdznB2j/thug-shakespeare.jpg');
+    background-size: cover;
+    color: #f8fae7;
+    font-weight: bold;
+  }
+
+  .display-1{
+    font-family: 'Courier New', Courier, monospace;
+    font-size: 4rem;
+    font-weight: 350;
+    line-height: 1.2;
+    text-shadow: 3px 3px #080808;
+  }
+
+  .display-2{
+      font-size: 2.25rem;
+      font-weight: 400;
+      line-height: 1;
+      text-shadow: 3px 3px #080808;
+  }
+
+  .separator{
+    border: 1.5px;
+    border-style:solid;
+  }
+</style>
+
+<body style="background-color:#434446;">
+</body>
```

Comment:
There's already a body element defined in layout html that this file's html gets nested inside.

What you've technically done is put an empty body inside the real body but HTML only allows one body tag so the browser may then fix up your HTML to what it thinks you meant. But it doesn't have to, some browsers may just fall over or ignore this.